### PR TITLE
Added Linux Support to Project

### PR DESCRIPTION
**Pull request by Bored0ne**
This commit rewrites a few functions and adds linux support to the project. If approved I also ask that you tag this and the Dollar project so that they can both be used on a linux system.

### CODE REVIEW

File: Sources/Regex.swift [hunk 1]
```diff
@@ -36,14 +39,14 @@ public class Regex {
     }
 
     public class func escapeStr(str: String) -> String {
-        let matches = RegexPatternRegex.matches(testStr: str)
+        let matches = Regex(RegexEscapePattern).matches(testStr: str)
```
Comment (ankurp) [inline Sources/Regex.swift]: Why use this instead of `RegexPatternRegex`?

**Comment by ankurp** [comment]
Thanks for the PR and will create a release once this is merged. Can you address my comment https://github.com/ankurp/Cent/pull/18/files/5c19b62077f28212e565122e05b8f053de198889#r84699225

**Comment by Bored0ne** [comment]
@ankurp In response to your comment, I thought it was causing a separate issue. It turns out to be perfectly fine the way it was.

**Comment by ankurp** [comment]
I squashed your commits and merging it as part of this PR. Will create release soon. https://github.com/ankurp/Cent/pull/19

**Comment by ankurp** [comment]
@Bored0ne This is now published and tagged as version 6.0.2

**Pull request closed without merging**